examples/eos-compose/eos-fst/setup

```diff
@@ -7,4 +7,5 @@ for i in {1..4}; do
   mkdir -p /disks/eosfs${i}
   chown daemon:daemon /disks/eosfs${i}
   eos -r 0 0 -b fs add eosfs${i} fst.testnet:1095 /disks/eosfs${i} default rw
+  eos -r 0 0 space set default on
```

Comment:
what is the reason for this?
And why in fst instead of mgm node?

Reply:
it looks like it does not make any difference where that command is run to enable the space, but its more about when its run

in eosmgm/setup the same command is executed and if I add a `sleep 30` before it, the spaces are enabled

to check do
`eos health -a `

```
┌────────────┬────────────┬────────────┬────────┐
│group       │offline used│ online free│  status│
└────────────┴────────────┴────────────┴────────┘
 default.0       781.49 GB          0 B     full 
```

vs.

```
┌────────────┬────────────┬────────────┬────────┐
│group       │offline used│ online free│  status│
└────────────┴────────────┴────────────┴────────┘
 default.0             0 B    213.10 GB       ok 
```

so I guess the enable command has to be run after the `fs add eosfs ...`